environment/src/main/java/com/sequenceiq/environment/parameters/dao/converter/ResourceTypeConverter.java

```diff
@@ -1,12 +1,12 @@
 package com.sequenceiq.environment.parameters.dao.converter;
 
 import com.sequenceiq.cloudbreak.converter.DefaultEnumConverter;
-import com.sequenceiq.environment.resourcepersister.ResourceType;
+import com.sequenceiq.common.api.type.ResourceType;
 
 public class ResourceTypeConverter extends DefaultEnumConverter<ResourceType> {
 
     @Override
     public ResourceType getDefault() {
-        return ResourceType.AWS_INSTANCE;
+        return ResourceType.AZURE_PRIVATE_DNS_ZONE;
```

Comment:
so this won't result in any unexpected result? why have you changed this and why to this one?

Reply:
Because this is the only resource type stored in the environment service now.